dashboard/test/controllers/rubrics_controller_test.rb

```diff
@@ -117,8 +117,26 @@ class RubricsControllerTest < ActionController::TestCase
 
     learning_goal1 = create :learning_goal, rubric: @rubric
     learning_goal2 = create :learning_goal, rubric: @rubric
-    ai_evaluation1 = create :learning_goal_ai_evaluation, learning_goal: learning_goal1, user: student, requester: @teacher, understanding: 1
-    ai_evaluation2 = create :learning_goal_ai_evaluation, learning_goal: learning_goal2, user: student, requester: @teacher,  understanding: 2
+    # TODO: save this to a variable and use it to create learning goal ai evaluations
+    rubric_ai_evaluation = create(
+      :rubric_ai_evaluation,
+      rubric: rubric,
+      user: student,
+      requester: @teacher,
+      status: 1
+    )
+    ai_evaluation1 = create(
+      :learning_goal_ai_evaluation,
+      rubric_ai_evaluation: rubric_ai_evaluation,
+      learning_goal: learning_goal1,
+      understanding: 1
+    )
+    ai_evaluation2 = create(
+      :learning_goal_ai_evaluation,
+      rubric_ai_evaluation: rubric_ai_evaluation,
+      learning_goal: learning_goal2,
+      understanding: 2
+    )
```

Comment:
does not need to be done now, but a `:with_ai_evaluations` would come in handy. see `:with_teacher_evaluations` for inspo: https://github.com/code-dot-org/code-dot-org/blob/4bec9a74e031fb8e08f8590f4de584f4f4b6482f/dashboard/test/factories/factories.rb#L1785-L1789

Reply:
Yep. That was my thought as well.